File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseSupportedLocations.scala

package org.openapitools.client.model


case class ResponseSupportedLocations (
    _locations: List[ResponseSupportedLocation],
    _unsupportedLocations: List[String]
)
object ResponseSupportedLocations {
    def toStringBody(var_locations: Object, var_unsupportedLocations: Object) =
        s"""
        | {
        | "locations":$var_locations,"unsupportedLocations":$var_unsupportedLocations
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseSupportedLocations.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseSupportedLocation

module ResponseSupportedLocations = 

  //#region ResponseSupportedLocations

  [<CLIMutable>]
  type ResponseSupportedLocations = {
    [<JsonProperty(PropertyName = "locations")>]
    Locations : ResponseSupportedLocation[];
    [<JsonProperty(PropertyName = "unsupported_locations")>]
    UnsupportedLocations : string[];
  }
  
  //#endregion
  